.claude/skills/nextjs-page-migration/SKILL.md

```
@@ -55,364 +75,109 @@ Phase A–F で 18 枚のガイドページが `web-next/` App Router に**全
 
-`globals.css` に実際に存在する変数（共有デザイントークン）:
-`--bg`, `--bg2`, `--srf`, `--srf2`, `--brd`, `--brd2`, `--txt`, `--txt2`, `--txt3`,
-`--acc`, `--acc2`, `--grn`, `--ylw`, `--red`, `--prp`, `--teal`, `--orng`
+#### ⚠️ CSS Module 地雷チェックリスト
 
-**② デスクトップサイドバーの固定には `position: sticky` を使い、モバイルには `position: fixed` を許可する**
+**① CSS変数は必ず `page.module.css` の最上位セレクタ内に定義する**
+`globals.css` に存在しない変数は必ず `.layout` / `.root` スコープに転写する。
 
+**② 全列左寄せアライメント原則**
 ```css
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)